src/uct/ib/efa/ib_efa_md.c

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include <uct/ib/efa/ib_efa.h>
+#include <uct/ib/base/ib_md.h>
```

Comment:
minor: it is already included in `<uct/ib/efa/ib_efa.h>`

Reply:
fixed